src/pages/books/index.jsx

```diff
@@ -0,0 +1,172 @@
+import React, { useEffect, useState } from 'react';
+import { Box, Typography, Link, useTheme } from '@mui/material';
+import { loadYamlData } from '../../utils/loadData';
+
+export default function BooksPage() {
+  const theme = useTheme();
+  const [books, setBooks] = useState([]);
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState(null);
+
+  useEffect(() => {
+    async function fetchBooks() {
+      try {
+        const bookFiles = [
+          'books/Hands-on_Pipeline_as_YAML_with_Jenkins.yml',
+          'books/integrating-php-projects-with-jenkins.yml',
+          'books/Jenkins_Essentials.yml',
+          'books/jenkins-the-definitive-guide-continuous_integration_for_the_masses.yml',
+          'books/jenkins2-up-and-running.yml',
+          'books/Learning_Continuous_Integration_with_Jenkins.yml',
+          'books/Mastering_Jenkins.yml'
+        ];
```

Comment:
We should not be hardcoding these. Could we please use something like `glob` (or a lightweight version of this) to fetch all files within a folder?